### PR TITLE
chore(ruby): Fix config file step (NR-39797)

### PR DESCRIPTION
**Pull request by austin-schaefer**
Fixes https://issues.newrelic.com/browse/NR-39797

### CODE REVIEW

File: src/content/docs/apm/agents/ruby-agent/installation/install-new-relic-ruby-agent.mdx [hunk 1]
```diff
@@ -88,16 +88,15 @@ Our Ruby agent auto-instruments your code so you can start monitoring applicatio
 
 After installing the agent, you will need to install the `newrelic.yml` configuration file, and name your app:
 
-1. If you haven't already, [create a New Relic account](https://newrelic.com/signup). It's free, forever. Or, [sign in](https://one.newrelic.com).
-2. From the [account dropdown](/docs/accounts-partnerships/education/getting-started-new-relic/glossary#account-dropdown), select **Account settings**.
-3. On the **Account settings** page, look for the **Download a clean configuration file** section and click the `newrelic.yml` file.
-4. Copy the `newrelic.yml` file into the `config` sub-directory of your application.
-5. [Change the default application name](/docs/agents/manage-apm-agents/app-naming/name-your-application) to a meaningful name.
+1. Download a copy of `[newrelic.yml](https://github.com/newrelic/newrelic-ruby-agent/blob/dev/newrelic.yml)` from our GitHub repo.
```
Comment (nbaenam) [inline src/content/docs/apm/agents/ruby-agent/installation/install-new-relic-ruby-agent.mdx:91]: The backtick shouldn't be at the end of the URL. I think it should be like this: `[newrelic.yml]`(URL)

File: src/content/docs/apm/agents/ruby-agent/installation/install-new-relic-ruby-agent.mdx [hunk 2]
```diff
@@ -88,16 +88,15 @@ Our Ruby agent auto-instruments your code so you can start monitoring applicatio
...
 After installing the agent, you will need to install the `newrelic.yml` configuration file, and name your app:
 
-1. If you haven't already, [create a New Relic account](https://newrelic.com/signup). It's free, forever. Or, [sign in](https://one.newrelic.com).
-2. From the [account dropdown](/docs/accounts-partnerships/education/getting-started-new-relic/glossary#account-dropdown), select **Account settings**.
-3. On the **Account settings** page, look for the **Download a clean configuration file** section and click the `newrelic.yml` file.
-4. Copy the `newrelic.yml` file into the `config` sub-directory of your application.
-5. [Change the default application name](/docs/agents/manage-apm-agents/app-naming/name-your-application) to a meaningful name.
+1. Download a copy of `[newrelic.yml](https://github.com/newrelic/newrelic-ruby-agent/blob/dev/newrelic.yml)` from our GitHub repo. 
+2. Copy the `newrelic.yml` file into the `config` sub-directory of your application.
+3. Edit the `license_key` value to include your license key ([Get your license key from the **API keys** page.](https://one.newrelic.com/launcher/api-keys-ui.api-keys-launcher).
```
Comment (nbaenam) [inline src/content/docs/apm/agents/ruby-agent/installation/install-new-relic-ruby-agent.mdx:93]: This step ends with 2 periods. You can see it better from the build.

File: src/content/docs/apm/agents/ruby-agent/installation/install-new-relic-ruby-agent.mdx (hunk 1 above)
Comment (austin-schaefer) [reply src/content/docs/apm/agents/ruby-agent/installation/install-new-relic-ruby-agent.mdx]: Yikes, I see what you mean. I'll try putting the backtick inside the link element.

File: src/content/docs/apm/agents/ruby-agent/installation/install-new-relic-ruby-agent.mdx (hunk 2 above)
Comment (austin-schaefer) [reply src/content/docs/apm/agents/ruby-agent/installation/install-new-relic-ruby-agent.mdx]: Good catch!

**Comment by austin-schaefer** [comment]
@nbaenam I think I fixed those two markup issues you caught, here's the latest deploy preview: https://docswebsitedevelop-austinschaeferpatch556490.gtsb.io/docs/apm/agents/ruby-agent/installation/install-new-relic-ruby-agent/#Configuration_file

Thanks again for your sharp eye on my poor markup!

**Review by nbaenam — APPROVED** [review]

**Pull request merged**